### PR TITLE
feat: add list component

### PR DESCRIPTION
**Pull request by nerdyspook**
**Added List Component**
> Simple List Groups
> Nested List Groups
> Spaced Inline List
> Notification Stacked List

lists

### CODE REVIEW

**Review by ishapatell1 — APPROVED** [review]
Looks good to merge!

File: components/lists/lists.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    color: var(--primary-color);
+}
+.list:last-child {
+    padding-bottom: 0;
+}
+
+.list li ul li {
+    color: var(--default-color);
+    margin: var(--sp-0-5) var(--sp-1);
+}
```
Comment (AmishMishra11) [inline components/lists/lists.css]: you could give this a class instead of doing this.

**Review by AmishMishra11 — COMMENTED** [review]
looks good just make this small change

File: components/lists/lists.css (hunk 1 above)
Comment (nerdyspook) [reply components/lists/lists.css]: Thanks, I have made the changes you suggested.

Comment (AmishMishra11) [reply components/lists/lists.css]: yep looks good now, you can merge it

**Pull request merged**